### PR TITLE
improved Hints paragraph in instructions

### PR DESCRIPTION
**Pull request by dawidsip**
The Hints encourage students to try out the Sprache library. But form what I've seen, very little have tried it. One couse of this might be how hard it is to find some TLDR help on Sprache. I believe with the addition of the blog I found the number of students to try Sprache should increase.

### CODE REVIEW

File: exercises/practice/wordy/.docs/instructions.append.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # Hints
 
-- To parse the text, you could try to use the [Sprache](https://github.com/sprache/Sprache/blob/develop/README.md) library. You can also find a good tutorial [here](https://www.thomaslevesque.com/2017/02/23/easy-text-parsing-in-c-with-sprache/).
+- To parse the text, you could try to use the [Sprache](https://github.com/sprache/Sprache/blob/develop/README.md) library. You can also find a good tutorial [here](https://www.thomaslevesque.com/2017/02/23/easy-text-parsing-in-c-with-sprache/), and a blog with useful hints and snippets over [here](https://justinpealing.me.uk/post/2020-04-20-sprache7-operators/)
```
Comment (ErikSchierboom) [inline exercises/practice/wordy/.docs/instructions.append.md:3]: Good idea. Any reason not to link to the first post in the blog? https://justinpealing.me.uk/post/2020-03-11-sprache1-chars/

Comment (dawidsip) [reply exercises/practice/wordy/.docs/instructions.append.md:3]: This is exactly where I also had daubts, should I give the link to the first post or rather the most relevant one. So I thought, the most relevant one is best, plus the blog always offers an Index section on the side, so it really isnt noticable which exact post is chosen at any time. But as soon as the student 'connects the dots' and realizes that there are arithmetic operations discussed in that exact post he clicked on it is more likely he'll try it out. This is my train of thought.

**Comment by ErikSchierboom** [comment]
Thanks!

File: exercises/practice/wordy/.docs/instructions.append.md (hunk 1 above)
Comment (dawidsip) [reply exercises/practice/wordy/.docs/instructions.append.md:3]: Right. Thank you for the quick reopening of that PR. I saw a hint to use Sprache in two more exercises(hard ones). I intend to solve them this month. If I find the tips in that blog also useful for those exercises I'll try to suggest a change to instructions of those as well.

**Pull request merged**